functions/package.json

```diff
@@ -13,6 +13,7 @@
     "cors": "2.8.4",
     "firebase-admin": "5.13.1",
     "firebase-functions": "2.0.2",
+    "lodash.partition": "^4.6.0",
```

Comment:
This project uses exact versions of dependencies instead of relying on semver ranges like this one. Remove the caret (`^`) to specify the exact version.